alpaca/broker/client.py

```diff
@@ -1763,7 +1763,7 @@ def get_trade_events(self, filter: Optional[GetEventsRequest] = None) -> Iterato
         if filter:
             params = filter.to_request_fields()
 
-        url = self._base_url + self._api_version + "/events/trades"
+        url = self._base_url + "/" + self._api_version + "/events/trades"
```

Comment:
Should we be using some sort of URL construction library here? If not, can we make this an f-string?